mmseg/models/backbones/vit.py

```diff
@@ -272,17 +272,9 @@ def init_weights(self):
                 self.pretrained, logger=logger, map_location='cpu')
             if 'state_dict' in checkpoint:
                 state_dict = checkpoint['state_dict']
-            elif 'model' in checkpoint:
-                state_dict = checkpoint['model']
             else:
                 state_dict = checkpoint
 
-            if self.pretrain_style == 'timm':
```

Comment:
The `pretrain_style` argument is not used, so it can be removed from `__init__`.